tox.ini

```diff
@@ -24,6 +24,14 @@ commands =
     sphinx-build -d "{toxworkdir}/docs_doctree" docs "{toxworkdir}/docs_out" --color -W -bhtml {posargs}
     python -c 'import pathlib; print("documentation available under file://\{0\}".format(pathlib.Path(r"{toxworkdir}") / "docs_out" / "index.html"))'
 
+# Binder
+[testenv:binder]
+description = execute all notebooks in binder to ensure they're runnable
```

Comment:
Should we change this to something like "execute all notebooks in the environment used on Binder'? When I read this I thought the test would be making calls to mybinder.org and we confused that it wasn't.